### ISSUE TITLE
Stacktrace when logging in

### ISSUE THREAD

**Issue by dennusb** [Graylog2/graylog2-web-interface#1343]
Hi,

I just installed beta3 and when i want to login i get this :
```
2015-05-28_12:07:17.88553 [error] play - Cannot invoke the action, eventually got an error: java.lang.RuntimeException: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.88568 [error] play - Cannot invoke the action, eventually got an error: java.lang.RuntimeException: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.88966 [error] application -
2015-05-28_12:07:17.88967
2015-05-28_12:07:17.88968 ! @6ma58n2jg - Internal server error, for (GET) [/] ->
2015-05-28_12:07:17.88968
2015-05-28_12:07:17.88968 play.api.Application$$anon$1: Execution exception[[RuntimeException: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])]]
2015-05-28_12:07:17.88970 	at play.api.Application$class.handleError(Application.scala:296) ~[com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.88970 	at play.api.DefaultApplication.handleError(Application.scala:402) [com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.88970 	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.88971 	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.88971 	at scala.Option.map(Option.scala:145) [org.scala-lang.scala-library-2.10.4.jar:na]
2015-05-28_12:07:17.88971 Caused by: java.lang.RuntimeException: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.88972 	at play.libs.Json.toJson(Json.java:35) ~[com.typesafe.play.play-json_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.88972 	at views.html.search.index$.apply(index.template.scala:67) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88972 	at views.html.search.index$.render(index.template.scala:77) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88972 	at views.html.search.index.render(index.template.scala) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88973 	at controllers.SearchController.renderSearch(SearchController.java:287) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88973 Caused by: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.88973 	at com.fasterxml.jackson.databind.ObjectMapper.valueToTree(ObjectMapper.java:2372) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.88974 	at play.libs.Json.toJson(Json.java:33) ~[com.typesafe.play.play-json_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.88974 	at views.html.search.index$.apply(index.template.scala:67) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88974 	at views.html.search.index$.render(index.template.scala:77) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88974 	at views.html.search.index.render(index.template.scala) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88975 Caused by: com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.88975 	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:210) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.88975 	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:177) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.88975 	at com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:190) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.88976 	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:674) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.88976 	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:156) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.88977 Caused by: java.lang.NullPointerException: null
2015-05-28_12:07:17.88977 	at models.descriptions.InputDescription.getCreatorUser(InputDescription.java:50) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.88977 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_40]
2015-05-28_12:07:17.88977 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_40]
2015-05-28_12:07:17.88978 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_40]
2015-05-28_12:07:17.88978 	at java.lang.reflect.Method.invoke(Method.java:497) ~[na:1.8.0_40]
2015-05-28_12:07:17.89167 [error] application -
2015-05-28_12:07:17.89168
2015-05-28_12:07:17.89169 ! @6ma58n2jg - Internal server error, for (GET) [/] ->
2015-05-28_12:07:17.89169
2015-05-28_12:07:17.89169 play.api.Application$$anon$1: Execution exception[[RuntimeException: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])]]
2015-05-28_12:07:17.89169 	at play.api.Application$class.handleError(Application.scala:296) ~[com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.89169 	at play.api.DefaultApplication.handleError(Application.scala:402) [com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.89170 	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.89170 	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$4.apply(PlayDefaultUpstreamHandler.scala:320) [com.typesafe.play.play_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.89170 	at scala.Option.map(Option.scala:145) [org.scala-lang.scala-library-2.10.4.jar:na]
2015-05-28_12:07:17.89171 Caused by: java.lang.RuntimeException: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.89171 	at play.libs.Json.toJson(Json.java:35) ~[com.typesafe.play.play-json_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.89171 	at views.html.search.index$.apply(index.template.scala:67) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89171 	at views.html.search.index$.render(index.template.scala:77) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89172 	at views.html.search.index.render(index.template.scala) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89172 	at controllers.SearchController.renderSearch(SearchController.java:287) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89172 Caused by: java.lang.IllegalArgumentException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.89173 	at com.fasterxml.jackson.databind.ObjectMapper.valueToTree(ObjectMapper.java:2372) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.89173 	at play.libs.Json.toJson(Json.java:33) ~[com.typesafe.play.play-json_2.10-2.3.9.jar:2.3.9]
2015-05-28_12:07:17.89173 	at views.html.search.index$.apply(index.template.scala:67) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89173 	at views.html.search.index$.render(index.template.scala:77) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89174 	at views.html.search.index.render(index.template.scala) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89175 Caused by: com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.NullPointerException) (through reference chain: com.google.common.collect.RegularImmutableMap["551010f8e4b0d49bcaa62d80"]->models.descriptions.InputDescription["creator_user"])
2015-05-28_12:07:17.89175 	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:210) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.89175 	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:177) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.89176 	at com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:190) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.89176 	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:674) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.89176 	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:156) ~[com.fasterxml.jackson.core.jackson-databind-2.5.3.jar:2.5.3]
2015-05-28_12:07:17.89176 Caused by: java.lang.NullPointerException: null
2015-05-28_12:07:17.89177 	at models.descriptions.InputDescription.getCreatorUser(InputDescription.java:50) ~[graylog-web-interface.graylog-web-interface-1.1.0-beta.2.jar:1.1.0-beta.2]
2015-05-28_12:07:17.89177 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_40]
2015-05-28_12:07:17.89177 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_40]
2015-05-28_12:07:17.89177 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_40]
2015-05-28_12:07:17.89178 	at java.lang.reflect.Method.invoke(Method.java:497) ~[na:1.8.0_40]
```
What can i do?

**Comment by kroepke** [comment]
This looks like you ran a beta.2 web interface against a beta.3 server. There have been incompatible changes, I guess :/

Can you verify it with both being on beta.3 please @dennusb

**Comment by joschi** [comment]
Duplicate of #1312 (and fixed in 1.1.0-beta.3).
